bot/seasons/easter/egg_hunt/cog.py

```diff
@@ -366,7 +379,7 @@ def current_timestamp() -> int:
                 await SuperEggMessage(msg, egg, current_window).start()
 
             log.debug("Sleeping until next window.")
-            next_loop = max(next_window - self.current_timestamp(), 0)
+            next_loop = max(next_window - self.current_timestamp(), 60*60)
```

Comment:
maybe these should be like class attribute constants instead of hardcoded values